### ISSUE TITLE
It can't see documented tasks in other files

### ISSUE THREAD

**Issue by fracz** [Mikhus/gulp-help-doc#2]
Given two gulpfiles:

_gulpfile.js_

``` js
var gulp = require('gulp');

require('./build/gulpfile-something');

gulp.task('help', function() {
    return require('gulp-help-doc')(gulp);
});
```

_build/gulpfile-something.js_

``` js
var gulp = require('gulp');
/**
 * Something
 * @task {sth}
 */
gulp.task('sth', function() {
});
```

The `gulp help` displays nothing.

**Comment by Mikhus** [comment]
True, currently this is unexpected behavior. Planned to be resolved within the next release.Thank you for contribution!

**Comment by Mikhus** [comment]
Fixed in release 1.0.6
